hhu-adam/lean4monaco
# fix: replace noto font with select glyphs

Tested on Mac + Linux.

Edit: Need to fix for Linux Chrome. Firefox works.

![image](https://github.com/user-attachments/assets/b19c169d-fcc0-4fb6-88ef-ae44a8e01c1a)

### Changes:
- replace Noto font with LeanWeb
- apply LeanWeb before JuliaMono in editor

### To test:
- run the demo
- make sure glyphs ✅, ❌, 🎉, ▼, ▶, 🎭 and ✝ render like with Noto font
- check \R still renders like JuliaMono font
